Directory.Build.props

```
@@ -5,7 +5,7 @@
     <Authors>$(Company)</Authors>
     <Copyright>Copyright © $(Company) $([System.DateTime]::Now.Year)</Copyright>
     <Trademark>$(Company)™</Trademark>
-    <VersionPrefix>1.0.1</VersionPrefix>
+    <VersionPrefix>1.0.2</VersionPrefix>
```

```suggestion
    <VersionPrefix>1.0.1</VersionPrefix>
```
We'll change the version number outside of the PR because we can always combine multiple PRs into a single release.